app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class UserMailer < ApplicationMailer
+  def welcome_email(group)
+    recipients = group.members.map(&:user)
+    mail(
+      from:'shahin@jann.com',
```

Comment:
I would change this to something like `noreply@tribal.com`

Reply:
updated